addOns/ascanrulesAlpha/src/main/resources/org/zaproxy/zap/extension/ascanrulesAlpha/resources/Messages.properties

```diff
@@ -50,3 +50,8 @@ ascanalpha.springactuator.desc=Spring Actuator for Health is enabled and may rev
 ascanalpha.springactuator.soln=Disable the Health Actuators and other actuators, or restrict them to administrative users.
 ascanalpha.springactuator.refs=https://docs.spring.io/spring-boot/docs/current/actuator-api/htmlsingle/#overview
 
+ascanalpha.log4shell.name=Log4Shell (CVE-2021-44228)
+ascanalpha.log4shell.desc=Apache Log4j2 <=2.14.1 JNDI features used in configuration, log messages, and parameters do not protect against attacker controlled LDAP and other JNDI related endpoints. An attacker who can control log messages or log message parameters can execute arbitrary code loaded from LDAP servers when message lookup substitution is enabled. From log4j 2.15.0, this behavior has been disabled by default.
+ascanalpha.log4shell.soln=Upgrade Log4j2 to version 2.15.0 or newer. In previous releases (>2.10) this behavior can be mitigated by setting system property "log4j2.formatMsgNoLookups" to "true" or by removing the JndiLookup class from the classpath (example: zip -q -d log4j-core-*.jar org/apache/logging/log4j/core/lookup/JndiLookup.class). Java 8u121 (see https://www.oracle.com/java/technologies/javase/8u121-relnotes.html) protects against remote code execution by defaulting "com.sun.jndi.rmi.object.trustURLCodebase" and "com.sun.jndi.cosnaming.object.trustURLCodebase" to "false".
+ascanalpha.log4shell.refs=https://www.cve.org/CVERecord?id=CVE-2021-44228\nhttps://www.lunasec.io/docs/blog/log4j-zero-day/\nhttps://nvd.nist.gov/vuln/detail/CVE-2021-44228
+ascanalpha.log4shell.skipped=Skipping Log4Shell Scan Rule because no Active Scan OAST service is selected.
```

Comment:
`no Active Scan OAST service is selected` should be enough, core already says the rule is being skipped.